astroquery/jplhorizons/core.py

```diff
@@ -809,7 +812,9 @@ def elements_async(self, get_query_payload=False,
 
     def vectors_async(self, get_query_payload=False,
                       closest_apparition=False, no_fragments=False,
-                      get_raw_response=False, cache=True, refplane='ecliptic'):
+                      refplane='ecliptic', aberrations='geometric',
```

Comment:
same here, keep kwarg order, add new ones to the end of the list

Reply:
Done, thanks!